.travis.yml

```diff
@@ -30,11 +30,9 @@ script:
   - make lint
   - make vet
   - if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then make semantic RANGE=$(scripts/travis_range.sh); fi
-  - make stress-test-async CONCURRENT_REQUESTS=256
```

Comment:
😞 

@ivanilves maybe move to CircleCI? Travis is such a piece of crap these days...

Reply:
I'll definitely think about this. I don't depend on any Travis-specific things.